examples/stocks/test_driver/scroll_perf.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_driver/driver_extension.dart';
+import 'package:flutter_driver/src/error.dart';
+import 'package:stocks/main.dart' as app;
+
+void main() {
```

Comment:
Eventually we'll move all driver tests under `test`, but today we are unable to tell them apart from unit-tests. For example, `flutter test` and `pub run test` will attempt to run a driver test, and will fail. Once we gain ability to use `@TestOn` we should be able to put all tests under `test` and run `pub run test -p flutter_driver` and have `flutter test` recognize them out-of-the-box.